src/clib/initialize_cloudy_data.c

```diff
@@ -313,3 +313,18 @@ int initialize_cloudy_data(chemistry_data *my_chemistry,
 
   return SUCCESS;
 }
+
+int free_cloudy_data(cloudy_data *my_cloudy, chemistry_data *my_chemistry, int primordial) {
+  for(int i = 0; i < my_cloudy->grid_rank; i++) {
```

Comment:
I think inline loop variable definitions are not always allowed in C. I think you just need to move the declaration of `i` out of the loop. I think this is why the tests are failing.

Reply:
I can do it.

The inline declaration was introduced in 1999, any recent compiler should be able to do it.

Reply:
Yes, true, I think it would also be acceptable to add the `-c99` flag to the make file. I think as long as the tests pass, it's ok.